main.tf

```diff
@@ -1,6 +1,7 @@
 locals {
   image_uri = "${var.ecr_repo_url}:${var.from_env}"
   artifacts_bucket_name = "s3-codepipeline-${var.app_name}-${var.env_type}"
+  run_tests = var.run_integration_tests || var.run_stress_tests ? true : false
```

Comment:
I think the conditional expression is redundant, you can make it simpler if the vars are boolean:
`run_tests = var.run_integration_tests || var.run_stress_tests`

Reply:
this is boolean, but the condition is so we can run only integration or only stress or both